plugin/src/generators/react-lib/generator.ts

```diff
@@ -2,7 +2,7 @@ import { formatFiles, generateFiles, Tree } from '@nx/devkit';
 import * as path from 'path';
 import { ReactLibGeneratorSchema } from './schema';
 import { execSync } from 'child_process';
-import { formatName, getLibName, askQuestion } from '../../shared/utils';
+import { formatName, getLibName, askQuestion } from '../../shared';
```

Comment:
Let's group all utils into a directory:

```suggestion
import { formatName, getLibName, askQuestion } from '../../shared/utils';
```

Reply:
done